<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DSGTools: Manage Complex Structures</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Database:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="dbCombo"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Complex Class:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="complexCombo"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="dbButton">
        <property name="text">
         <string>Load DB's</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="managePushButton">
        <property name="text">
         <string>Manage</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="associatePushButton">
        <property name="text">
         <string>Associate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="disassociatePushButton">
        <property name="text">
         <string>Disassociate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="zoomButton">
        <property name="text">
         <string>Zoom</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QTreeWidget" name="treeWidget">
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Value</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
